other.yml

```diff
@@ -18,6 +18,8 @@ events:
       until: 2023-06-30  # required
```

Comment:
Suggest: 2023-03-31.

Reply:
I'm not adding a new series. I'm adding a onetime Lean Coffee (but that should probably go to the onetimes file then - I'll fix that).